uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/io/HabitBullCSVImporter.kt

```diff
@@ -77,7 +77,8 @@ class HabitBullCSVImporter
                 logger.info("Creating habit: $name")
             }
             if (parseInt(cols[4]) == 1) {
-                h.originalEntries.add(Entry(timestamp, Entry.YES_MANUAL))
+                val notes = cols[5] ?: ""
+                h.originalEntries.add(Entry(timestamp, Entry.YES_MANUAL, notes))
```

Comment:
I think there are test cases for the Habit Bull importer, could you they be expanded to check if it works with the notes as well?